server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -4900,3 +4900,26 @@ This is an example of returned body.
   "targetsContent": false
 }
 ....
+
+== Reloading server certificates
+
+Certificates for TCP based protocols (IMAP, SMTP, POP3, LMTP and ManageSieve) can be updated at
+runtime, without service interuption and without closing existing connections.
+
+In order to do so:
+
+ - Generate / retrieve your cryptographic materials and replace the ones specified in James configuration.
+ - Then call the following endpoint:
+
+....
+curl -XPOST http://ip:port/servers?reload-certificate
+....
+
+Optional query parameters:
+
+ - `port`: positive integer (valid port number). Only reload certificates for the specific port.
```

Comment:
I find a bit weird to use the port here, can't we bind several different configurations to different IPs?

Reply:
I would have prefered a server name but there is no way to easily access such a thing.

Also binding same port multiple time for different IPs is a non default advanced configuration that 99% of the users won't use, and the port number would server them well.